materials/using-gem5/03-gem5-basics/02-frequently-used-features/04-list-comprehension.py

```diff
@@ -0,0 +1,42 @@
+from pprint import pprint
+from gem5.components.processors.simple_processor import SimpleProcessor
+from gem5.components.processors.cpu_types import CPUTypes
+from gem5.isas import ISA
+
+if __name__ == "__m5_main__":
+    print("List Comprehension")
+
+    print("Example 1")
+    x = [k for k in range(10)]  # typical list comprehension
+    print("x =", x)
```

Comment:
Use f-strings
```suggestion
    print(f"x = {x}")
```
Unfortunately, we can't guarantee the most recent versions of python (3.9+?) where you could use `print(f"{x=}")` or something like that to get the same result.

Reply:
I'd prefer not to use `print(f"{x=}")` because it is quite confusing